ajax/libs/angularjs-nvd3-directives/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "angularjs-nvd3-directives",
+  "version": "0.0.7",
+  "filename": "angularjs-nvd3-directives.min.js",
+  "homepage": "http://cmaurer.github.io/angularjs-nvd3-directives",
+  "description": "Angular.js directives for nvd3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cmaurer/angularjs-nvd3-directives.git"
```

Comment:
`https://github.com/cmaurer/angularjs-nvd3-directives` will redirect to
`https://github.com/angularjs-nvd3-directives/angularjs-nvd3-directives` now.

Could you make `repository` & `autoupdate` field use the same repo?